README.md

```diff
@@ -24,7 +24,7 @@ This is available for an Intel based System (e.g. Synology Docker) as well as an
 ## Running docker
 Choose for the fitting docker tag and run the server with the following parameters:
 
-```sudo docker run -p 3000:3000 --mount type=bind,source=/PATH_TO_LOCAL_CONFIG,target=/config --mount type=bind,source=/PATH_TO_LOCAL_LOG,target=/log jomjol/wasserzaehler:DOCKER_TAG```
```

Comment:
If you have to use sudo to execute docker, your user is not member of the `docker` group